### PR TITLE
alert and button components

### PR DESCRIPTION
**Pull request by rohanmathur91**
Please review the alert and button components.
image
image

### CODE REVIEW

**Review by HarshithaKonduru214 — APPROVED** [review]

File: alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./alert.css" />
```
Comment (smilegupta) [inline alert/alert.html]: Just one suggestion, what you can do that is, you can maintain one single module like a main.css and you can include this all css files in it and say you are using font-awesome. Include it in one place of the project

File: alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (smilegupta) [inline alert/alert.html:6]: You can always include your name in the project meta tag just to give a personalised touch.

File: button/button.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+.btn-primary {
+	color: var(--color-white);
+	background-color: var(--color-blue-800);
```
Comment (smilegupta) [inline button/button.css]: Add name colors as more readable. color-blue-800 looks less human readable. If its a dark shade you can mention it as dark blue or some think that

File: button/button.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+.btn-outlined {
+	color: var(--color-blue-800);
+	border: 1px solid var(--color-blue-800);
+}
+
+.btn-outlined:hover {
+	color: var(--color-white);
+	background-color: var(--color-blue-800);
+}
+
```
Comment (smilegupta) [inline button/button.css]: Same suggestion as above, plus we know its a color.. you can emit the work color-white... just white is also ok...

you can mention color with like primary color or font-color something like this variables

**Review by smilegupta — APPROVED** [review]
Looks good to me. You can do some small changes if reqired. Rest looks good.

File: alert/alert.html (hunk 1 above)
Comment (rohanmathur91) [reply alert/alert.html]: Yes, I am going to do that only, this is only for review purposes, after that, I will put all my components CSS in one file and will import in the main styles.css.

File: alert/alert.html (hunk 2 above)
Comment (rohanmathur91) [reply alert/alert.html:6]: Will do that, thanks for the suggestion.

File: button/button.css (hunk 3 above)
Comment (rohanmathur91) [reply button/button.css]: Changes done

File: button/button.css (hunk 4 above)
Comment (rohanmathur91) [reply button/button.css]: Changes done.

**Review by smilegupta — APPROVED** [review]
LGTM!

**Pull request closed without merging**